docs/shared-content-source/docs/modules/ROOT/pages/app-development-process.adoc

```diff
@@ -27,7 +27,9 @@ The Streamlet model offers an abstraction to:
 
 In the current version, Cloudflow includes backend `Streamlet` implementations for Akka, Apache Spark - Structured Streaming, and Apache Flink. 
 Using these implementations you can write business logic in the native API of the backend.
-Additionally, Cloudflow can be extended with new streaming backends.
+Along with the currently supported built-in integration of Flink and Spark streamlets via the Cloudflow operator, Cloudflow also supports external integrations for these streaming platforms through additional plugins. The current version introduces Flink and Spark integrations with more controls in the hands of the users. Users now have more control on deployment and management of Flink and Spark streamlets while still using the same Cloudflow streamlet API for developing their business logic. However, Akka will continue to be supported natively as in earlier versions.
```

Comment:
You can remove:
```
The current version introduces Flink and Spark integrations with more controls in the hands of the users.
```

And start the following sentence with something like:
```
Using this new integration ...
```

Reply:
done ..